libobs/util/platform-nix.c

```diff
@@ -392,7 +392,8 @@ struct os_dirent *os_readdir(os_dir_t *dir)
 	if (!dir->cur_dirent)
 		return NULL;
 
-	strncpy(dir->out.d_name, dir->cur_dirent->d_name, 255);
+	strncpy(dir->out.d_name, dir->cur_dirent->d_name,
```

Comment:
This is intentional to ensure there is a null at the end of the string.

Reply:
Alright, will drop this commit